.golangci.yaml

```diff
@@ -34,3 +35,7 @@ linters-settings:
     statements: 48
   lll:
     line-length: 100
+  revive:
+    rules:
+      - name: var-naming
+        arguments: [["UID", "GID"]]
```

Comment:
Can this consume our existing revive config file rather than reproducing the settings here?

Reply:
Apparently, there isn't such option. I merged the rest of revive's toml config-options into golangci.yaml

Reply:
OK, that's a bit disappointing to hear. I won't object to it in this PR but I will reserve the right to do something about the duplication in the future if keeping these configs in sync becomes burdensome.

Hopefully before that happens someone (us?) could contribute to the linter the ability to pass a path :-)